tablechop

```diff
@@ -75,25 +75,27 @@ def clean_backups(args, log):
                 break
         ky = bucket.get_key(jkey)
         jdict = json.loads(ky.get_contents_as_string())
-        if len(jdict.values()) != 1:
+        if len(list(jdict.values())) != 1:
             raise SystemError('-listdir.json file should have '
                 'a single key/value pair!')
-        dirname = jdict.keys()[0]
+        dirname = list(jdict.keys())[0]
         # fullpaths here since some files are in subdirectories
-        fullpaths = [os.path.join(dirname, x) for x in jdict.values()[0]]
+        fullpaths = [os.path.join(dirname, x) for x in list(jdict.values())[0]]
         keep_jkey = False # assume we are deleting the jkey
         for x in fullpaths:
-	    keep_file = True # assume we are keeping the file - this is the default behavior
-	    # if we are checking if we should keep existing files
-	    if args.keep_existing and is_old and not os.path.isfile(x):
-	        # don't keep the file if it is old and it does not exist
-	        keep_file = False
-
-            keep_jkey = keep_jkey or keep_file  # keep the jkey if we are keeping any of the files
-	    key_to_keep = '%s:%s' % (args.name, x)
-	    # if we are keeping the file
-	    if keep_file and key_to_keep in to_delete:
-	        to_delete.remove(key_to_keep)
+            keep_file = True # assume we are keeping the file - this is the
+            # default behavior
+            # if we are checking if we should keep existing files
```

Comment:
Did the `2to3` tool split this comment over two lines?

Reply:
Maybe PyCharm or Atom did. Moved & combined 2 comment lines into 1.

Reply:
OK, confirmed conversion to just one comment on the previous line. Looks good.